scapy/sendrecv.py

```diff
@@ -558,47 +558,60 @@ def sniff(count=0, store=1, offline=None, prn = None, lfilter=None, L2socket=Non
 stop_filter: python function applied to each packet to determine
              if we have to stop the capture after this packet
              ex: stop_filter = lambda x: x.haslayer(TCP)
+iface: interface or list of interfaces (default: None for sniffing on all interfaces)
     """
     c = 0
-    
+    label={}
+    ls=[] 
     if opened_socket is not None:
-        s = opened_socket
+        ls = [opened_socket]
     else:
         if offline is None:
             if L2socket is None:
                 L2socket = conf.L2listen
-            s = L2socket(type=ETH_P_ALL, *arg, **karg)
+            if type(iface) is list:
+                for i in iface:
+                    s=L2socket(type=ETH_P_ALL, iface=i, *arg, **karg) 
```

Comment:
It will be nice if you can PEP08'ify your patch. The whole Scapy code is far from being compliant, but we try to improve that along new patches.